torch/distributed/_tensor/dispatch.py

```diff
@@ -55,6 +56,64 @@ def is_same_size_handler(
     return lhs.shape == rhs.shape
 
 
+def convolution_handler(
+    op_call: torch._ops.OpOverload,
+    args: Tuple[object, ...],
+    kwargs: Dict[str, object],
+) -> object:
+    # extract local tensor and sharding infos to a OpInfo
+    op_info = dtensor.DTensor._op_dispatcher.unwrap_to_op_info(op_call, args, kwargs)
+
+    # sharding propagation
+    dtensor.DTensor._op_dispatcher.sharding_propagator.propagate(op_info)
+    output_sharding = op_info.output_sharding
+    assert output_sharding is not None, "output sharding should not be None"
+
+    # local propagation
+    local_tensor_args = [
```

Comment:
local_tensor_args can be accessed directly from `op_info`, i.e. op_info.local_args.

Reply:
Thanks for letting me know. I have done it.